src/brevitas/nn/equalized_layer.py

```diff
@@ -10,8 +10,20 @@ def __init__(self, scale_module, layer) -> None:
         self.scale = scale_module
         self.layer = layer
 
-    def forward(self, x, *args, **kwargs):
+    def forward(self, *args, **kwargs):
         args = list(args)
+
+        if len(args) > 0:
```

Comment:
We should probably standardize this better, in case of a kwargs with different names maybe.